packages/js/src/utils/errors.ts

```diff
@@ -13,6 +13,10 @@ const logtoErrorCodes = Object.freeze({
     state_mismatched: 'State mismatched',
     missing_code: 'Missing code',
   },
+  requester: {
+    not_ok: 'Not ok',
+    not_provide_fetch: 'Should provide a fetch function under Node.js',
```

Comment:
I don't quite get it, why do we have to emphasize "under Node.js"?
And let's put grammar aside, `A fetch function should be provided / is required` should be enough, IMO.

Reply:
Under browser environments, we use `window.fetch` to fetch response by default.
Under Node.js environments, the developers can use `node-fetch`, `got` or else.
Not provide default implementation under Node.js environments.
( follow the default implementation from @wangsijie )

Need further discussion.

Reply:
OK, I think I see your points. We can discuss it offline tomorrow

Reply:
Discussed, no problem for now. 
The current implementation requires the end user to specify a fetch function implementation in node.js environment. But for most of the cases, the environment would be the browser, so browser "fetch" function will be used by default.